shell/mixins/advanced-filtering.js

```diff
@@ -0,0 +1,253 @@
+import { ADV_FILTER_ALL_COLS_VALUE, ADV_FILTER_ALL_COLS_LABEL } from '@shell/components/SortableTable/filtering';
```

Comment:
Mixins for the SortableTable are just kept in the same `SortableTable` folder

Reply:
correct. moved